src/change_stream.ts

```diff
@@ -934,10 +934,14 @@ export class ChangeStream<
       this.cursor.close().catch(() => null);
 
       const topology = getTopology(this.parent);
-      topology.selectServer(this.cursor.readPreference, {}, serverSelectionError => {
-        if (serverSelectionError) return this._closeEmitterModeWithError(changeStreamError);
-        this.cursor = this._createChangeStreamCursor(this.cursor.resumeOptions);
-      });
+      topology.selectServer(
+        this.cursor.readPreference,
+        { operationName: 'reconnect topology' },
```

Comment:
Which operations cause a server selection? Should this be the particular name of the operation for which a server is being selected?

Reply:
The spec says that operation is `The name of the operation for which a server is being selected. When server selection is being performed to select a server for a command, this MUST be the command name`. If different commands cause server selection, is there a way to pass the actual command name to this method?

Reply:
There's not a specific command name for iterating through the change stream as far as I'm aware. The reason for this server selection is that while iterating through the change stream, connection was lost. This is why I named it "reconnect topology." We could very well change it to something like "reconnect topology in change stream" to be more descriptive.

Reply:
In this case yes, the "reconnect topology in change stream" name or something like this would be more helpful to read which command the server selection was performed to.

Reply:
This issue hasn't been resolved, right? I see the same command name here.

Reply:
Apologies for the late update, rebasing right now and will update with the new name.